packages/wrangler/src/dev.tsx

```diff
@@ -374,6 +374,7 @@ function useEsbuild(props: {
         ...(jsxFactory && { jsxFactory }),
         ...(jsxFragment && { jsxFragment }),
         external: ["__STATIC_CONTENT_MANIFEST"],
+        conditions: ["worker", "browser", "module", "imports", "default"],
```

Comment:
As per [esbuild docs](https://esbuild.github.io/api/#how-conditions-work):
- default, import, require, browser, and node conditions cannot be disabled - so using all of them here is redundant.  I'm not sure what `platform` is configured (if any) - so I think it's worth using browser condition explicitly here:
```suggestion
        conditions: ["worker", "browser"],
```
- the "imports" condition seems like a typo~ - the correct one would be "import' but, as mentioned above, this is redundant anyway